packages/backend/src/graphql/resolvers/webhooks.ts

```diff
@@ -0,0 +1,42 @@
+import { ApolloContext } from '../../app'
+import {
+  QueryResolvers,
+  ResolversTypes,
+  WebhookEvent as SchemaWebhookEvent
+} from '../generated/graphql'
+import { getPageInfo } from '../../shared/pagination'
+import { WebhookEvent } from '../../webhook/model'
+import { Pagination } from '../../shared/baseModel'
+
+export const getWebhookEvents: QueryResolvers<ApolloContext>['webhookEvents'] =
+  async (
+    parent,
+    args,
+    ctx
+  ): Promise<ResolversTypes['WebhookEventsConnection']> => {
+    const { filter, ...pagination } = args
+    const webhookService = await ctx.container.use('webhookService')
+    const webhookEvents = await webhookService.getPage({ pagination, filter })
+    // TODO: test this... does getPageInfo work with filters?
+    const pageInfo = await getPageInfo(
+      (pagination: Pagination) =>
+        webhookService.getPage({ pagination, filter }),
+      webhookEvents
+    )
```

Comment:
Still need to implement the `getPageInfo` tests for this and verify that they pass (I think they should) with the filters and meaningfully test the filters.